### DISCUSSION TITLE
Inclusion of handleDidPartiallyAcceptCompletionItem in InlineCompletionItemProvider Interface to Stable API

### DISCUSSION THREAD

**Discussion by kangsan0420** [microsoft/vscode-discussions#834] [Extension Development QnA] [answered]
It appears that handleDidPartiallyAcceptCompletionItem in the InlineCompletionItemProvider interface is still a Proposed API.
When is it scheduled to be incorporated into the stable API?

**Comment by isidorn** [comment, marked as answer]
@kangsan0420 what is your use case for this API, and what scenario are you solving with it?

This API is currently in proposed state, and as with all API we have our finalization process which might take some time. If you provide feedback on the API that might help us finalize it sooner. Thanks!
